Classes/common/CDTDatastore.m

```diff
@@ -63,6 +64,7 @@ - (id)initWithDatabase:(TD_Database *)database
         NSString *name = [database name];
         _extensionsDir =
             [dir stringByAppendingPathComponent:[NSString stringWithFormat:@"%@_extensions", name]];
+        _manager = manager;
```

Comment:
it shouldn't be `nil` but I've opted not to validate it since CDTDatastore objects should only be created from CDTDatastoreManager